src/ucp/rndv/proto_rndv.c

```diff
@@ -439,8 +439,6 @@ ucp_proto_rndv_send_reply(ucp_worker_h worker, ucp_request_t *req,
                   ucp_operation_names[op_id], req->send.rndv.remote_address,
                   req->send.rndv.remote_req_id,
                   req->send.proto_config->proto->name);
-
-    ucp_request_send(req);
```

Comment:
why removed?

Reply:
it was moved outside this function, so could set req->recv.proto_rndv_config before calling `ucp_request_send(req)` (which may complete and release the request)

Reply:
ok, ic